test/parallel/test-uv-errno.js

```diff
@@ -14,20 +14,22 @@ const keys = Object.keys(uv);
 
 assert.strictEqual(uv.errname(-111111), 'Unknown system error -111111');
 
-keys.forEach((key) => {
+for (const key of keys) {
   if (!key.startsWith('UV_'))
-    return;
+    continue;
 
   const err = _errnoException(uv[key], 'test');
   const name = uv.errname(uv[key]);
   assert.strictEqual(getSystemErrorName(uv[key]), name);
   assert.strictEqual(err.code, name);
   assert.strictEqual(err.code, getSystemErrorName(err.errno));
   assert.strictEqual(err.message, `test ${name}`);
-});
+}
 
 function runTest(fn) {
-  ['test', {}, []].forEach((err) => {
+  const invalidArgType = ['test', {}, []];
+  const outOfRangeArgs = [0, 1, Infinity, -Infinity, NaN];
```

Comment:
it would be nice to move this line to line 42 so that the `outOfRangeArgs` declaration is right next to its usage.